blog/2024-01-30-FAQ.md

```diff
@@ -95,6 +95,23 @@ You can close your position, but your channel will remain open until you decide
 
 Depending on the size of this channel, you'll be able to open larger or smaller positions in the trading section, or hold more or less synthetic currency in your wallet (to be released)
 
+### Why my position has an expiry?
+
+This is how futures contracts work, and so are discreet log contracts.
+
+A position has an expiry of 1 week.
```

Comment:
I think this depends on when you open it. AFAIK, the position will simply expire on Sunday like you mention elsewhere.